src/main/java/nextstep/session/domain/Session.java

```diff
@@ -14,24 +15,26 @@ public class Session {
 
     private final LocalDateTime endedAt;
 
-    private final SessionCover cover;
+    private final List<SessionCover> covers;
 
     private final SessionType sessionType;
 
     private final Enrollment enrollment;
 
     public Session(Long id) {
-        this(id, null, null, null, null, null, null, null);
+        this(id, null, null, List.of(), null, null, null, null, null);
```

Comment:
```suggestion
        this(id, null, null, Collections.emptyList(), null, null, null, null, null);
```
이걸 사용하는게 더 좋지 않을까 생각이 들긴 하네요!

혹은 상태의 default값으로 빈 리스트를 선언해두는것도 방법일거 같습니다!